src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java

```diff
@@ -15,6 +15,7 @@
 public class ApplicationController {
```

Comment:
Please provide following for this class:
- Header
- Javadoc comment for each method